packages/react-theme/src/global/colors.ts

```diff
@@ -58,6 +58,32 @@ export const white = '#ffffff';
 
 export const black = '#000000';
 
+export const whiteAlpha = {
+  '5': 'rgba(255, 255, 255, 0.05)',
+  '10': 'rgba(255, 255, 255, 0.1)',
+  '20': 'rgba(255, 255, 255, 0.2)',
+  '30': 'rgba(255, 255, 255, 0.3)',
+  '40': 'rgba(255, 255, 255, 0.4)',
+  '50': 'rgba(255, 255, 255, 0.5)',
+  '60': 'rgba(255, 255, 255, 0.6)',
+  '70': 'rgba(255, 255, 255, 0.7)',
+  '80': 'rgba(255, 255, 255, 0.8)',
+  '90': 'rgba(255, 255, 255, 0.9)',
+};
+
+export const blackAlpha = {
```

Comment:
Let's type these so that accidental widening does not happen: 
[TS playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAggNmAFgQwMIHs7oE4GcoC8UArFAD5QCMADOVAEy0UDMTUALG8WwGxsDsbABxsAnNQDcAKCkBjdADtcwKAHdEAS2AR4SZISgBvKVCgByYmYBc57AHMARsgAU9YsQA0Dd17eeo1AB01MQAlGYeJuY01raOLn6+Pt7+QZThkaZmjLFm9k6uyYkpXkH0GVFmrLn5CUX1qYHMFVmcNfGF-sXFQewt5tztBd0NpYFhEZV8Q3VdowGBPP1mgjOdSXON-MsiayObY0LL4nvzPYGiFZUAZujouU7YkwC+0lKgkFAAKogQALbQIjGUzqLQ6BAoGwAJQg8mwABMADy6FAYLB4LzKbAaBR2AB80meMnkShUwF+AJsP3+gKMUVB2hRyEiRKkQA)

```suggestion
export const blackAlpha: Record<AlphaColors, string> = {
```